GDJS/Runtime/websocket-debugger-client/websocket-debugger-client.js

```diff
@@ -1,14 +1,64 @@
 /**
- * This DebuggerClient connects to a websocket server and can dump
+ * An client side implementation of the Debugger
+ * @interface
+ */
+gdjs.DebuggerClient = function() {} // Necessary for Interface definition 
+
+/**
+ * Update a value of the {@link RuntimeGame} instance.
+ * @function
+ * @name gdjs.DebuggerClient#set
+ * @param {Array<string>} path - The path to modify.
+ * @param {any} newValue - The new Value.
+ */
+
+/**
+ * Call a method of the {@link RuntimeGame} instance.
```

Comment:
Let's be a bit more precise, as we're not necessarily calling a runtime game method:
`Call a method, specified by a path starting from the {@link RuntimeGame} instance.`